src/ResourceManager/ApiManagement/Commands.ApiManagement.ServiceManagement/Commands/AzureApiManagementCmdletBase.cs

```diff
@@ -65,5 +68,64 @@ protected virtual void HandleException(Exception ex)
         {
             WriteExceptionError(ex);
         }
+        
+        protected void WriteProgress(TenantConfigurationLongRunningOperation operation)
+        {
+            WriteProgress(new ProgressRecord(0, operation.OperationName, operation.Status.ToString()));
+        }
+
+        protected TenantConfigurationLongRunningOperation WaitForOperationToComplete(TenantConfigurationLongRunningOperation longRunningOperation)
```

Comment:
Its a service side Code bug, I will add a TODO to revert to standard long running operation code